src/main/scala/thurloe/notification/NotificationMonitor.scala

```diff
@@ -423,6 +423,13 @@ class NotificationMonitorActor(val pollInterval: FiniteDuration,
           Map("originEmail" -> requesterId),
           Map("userNameFL" -> requesterId)
         )
+
+      case SnapshotReadyNotification(recipientUserId, snapshotExportLink, snapshotName, snapshotSummary) =>
+        thurloe.service.Notification(Option(recipientUserId), None, None, templateId, Map(
+          "snapshotExportLink" -> snapshotExportLink,
+          "snapshotName" -> snapshotName,
+          "snapshotSummary" -> snapshotSummary
+        ), Map.empty, Map.empty);
```

Comment:
thanks for walking through this with me. the map here represents substitutions to be made in the body of the email. the empty maps here represent substitutions for a user's email and name. We get the first name for free from the user id, so we don't need that substitution and we don't use the email in the body of the email, so we don't need that substitution.